docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::warning
+This implementation is experimental and subject to change.
+:::
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](./split-keyboards.md) with the dongle as "central". There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- The "central" dongle results in increased battery life for the keyboard/former "central", as it is now a peripheral.
+- It is easier to connect to the host device.
+
+Disadvantages:
+
+- An extra microcontroller is needed.
+- The keyboard becomes unusable without the dongle.
+
+
+
+## Editing a Keyboard Definition to Support a Dongle
+
+
+In the examples below, we will be modifying the [New Keyboard Shield](../development/new-shield.mdx) guide files and will be building on top of that.
+
+We will add the dongle in a way that it can be easily enabled, but is disabled by default.
+
+### Keyboard Shield
```

Comment:
```suggestion
### Shield Configuration Files
```
I think separating the sections by file type rather than by file makes more sense, since then you can write which file should be edited more clearly in the text.